### ISSUE TITLE
Voting functionality?

### ISSUE THREAD

**Issue by ErichDonGubler** [ErichDonGubler/not-yet-awesome-rust#2]
Would it even be possible to have a Github-auth'd voting button?

**Comment by lasizoillo** [comment]
You can order issues by thumbups reactions as quick'n'dirty voting system ;-)

**Comment by ErichDonGubler** [comment]
@lasizoillo: I wish I felt like I could use issues for this! I'm hoping to actually implement voting inside the list itself, so that passers-by can simply click a button and indicate their interest. However, I'd prefer it to be Github-auth'd, which makes things a little more sticky.

Does the new title of this issue clarify that? What do you think?

**Comment by lasizoillo** [comment]
Yes. It's clearer. I don't know any service to do that. Maybe must be created.

**Comment by ErichDonGubler** [comment]
I think I'm going to resolve this by putting this repo into its own organization and using another repo to keep discussion and voting separate from content requests. I will mark this issue as closed once this has been implemented!

**Comment by ErichDonGubler** [comment]
@jaroslaw-weber: This actually looks pretty close to what I was thinking! My hopeful vision of what the flow will look like also has a little widget that can be used to embed a button for voting with an HTML snippet...does that look doable with the Github API?

**Comment by jaroslaw-weber** [comment]
@ErichDonGubler 
Yes, being able to vote on the website would be great. 
I think it may be possible  but you would have to implement OAuth, and make a dynamic site and pay for the server (I am not a web dev so I don't know if it's possible with a static page).
Currently, I host the site on the GitHub pages so it's free and doesn't require much maintenance.
